### ISSUE TITLE
Explanations of examples

### ISSUE THREAD

**Issue by yang** [davidkpiano/flipping#11]
I've already been leveraging flipping.js for basic transitions, but the examples are neat! Any elaboration to help understand how they work / how they use flipping.js would be amazing.

**Comment by davidkpiano** [comment]
Yep, these are coming soon! I have lots of documentation plans for Flipping that I haven't gotten around to yet.
